tests/scripts/test_answerstore.py

```diff
@@ -312,3 +312,13 @@ def test_answerfile_generate(tmpdir, answerfile):
     assert store2['scope2']['key2'] == 'generate.scope2.key2'
     assert store2['boolscope']['key1'] is False
     assert store2['boolscope']['key2'] is True
+
+
+def test__comment_out():
+    text = 'A really long line of text.\nYep, with newlines.\n\nCouple of them.'
+    expected_text = ('# inikey:             A really long line of text.\n'
+                     '#                     Yep, with newlines.\n'
+                     '#                     Couple of them.')
+    key = 'inikey'
+    commented_text = _comment_out(key, text)
+    assert commented_text == expected_text
```

Comment:
add test for the string without the newline. current solution could crash if I see right.

Reply:
No it won't crash :) Test added.

Reply:
you are right! :) I though the behaviour is different in python for `[][5:]` :D nice to know!